.cirrus.yml

```diff
@@ -28,7 +28,7 @@ check_android_task:
     curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
     chmod +x ./cc-test-reporter
     ./cc-test-reporter before-build
-  check_script: ./gradlew check connectedCheck
```

Comment:
Is it necessary to change this in the main if it slows tests down ?

Reply:
It helps to find errors in case some tests don't pass, so I don't think it will pose problem, we will just have to begin reviewing and merging a bit earlier

Reply:
does it slow the tests down ?